cvat-sdk/cvat_sdk/core/proxies/tasks.py

```diff
@@ -119,6 +121,35 @@ def upload_data(
                 url, list(map(Path, resources)), pbar=pbar, **data
             )
 
+        if wait_for_completion:
+            if status_check_period is None:
+                status_check_period = self._client.config.status_check_period
+
+            self._client.logger.info("Awaiting for task %s creation...", self.id)
+            status: models.RqStatus = None
+            while status != models.RqStatusStateEnum.allowed_values[("value",)]["FINISHED"]:
+                sleep(status_check_period)
+                (status, response) = self.api.retrieve_status(self.id)
+
+                self._client.logger.info(
+                    "Task %s creation status=%s, message=%s",
```

Comment:
```suggestion
                    "Task %s creation state=%s, message=%s",
```

Reply:
I don't see how this improves the message.

Reply:
The actual field that is printed is called `state`. It's a matter of consistency.

Reply:
I'd vote for renaming the `state` into `name` then instead, because it just represents the status name.

Reply:
Maybe, but that seems like an excessive change for this PR.

How about this: `Task %s creation status: %s (message=%s)`? That makes it clear that both values are part of the status.

Reply:
Sounds good.